promptolution/optimizers/__init__.py

```diff
@@ -14,19 +14,24 @@
 from .opro import Opro
 
 
-def get_optimizer(config, *args, **kwargs):
+def get_optimizer(
+    config=None, optimizer: str = None, include_task_desc: bool = None, meta_prompt: str = None, *args, **kwargs
+):
     """Factory function to create and return an optimizer instance based on the provided configuration.
```

Comment:
add to comment that there are two possibilities: via config or via specification